includes/class-wc-bfx-pay-gateway.php

```diff
@@ -156,6 +155,28 @@ protected function setup_properties()
         $this->has_fields = true;
     }
 
+    protected function update_order_status($order, $payment_status)
+    {
+        if ('COMPLETED' === $payment_status) {
+            ob_start();
+            $order->payment_complete();
+            ob_clean();
+            return true;
+        }
+
+        if ('PENDING' === $payment_status) {
+            $order->update_status('on-hold');
+            return false;
```

Comment:
hmm returning false here is a bit confusing, I think we should return false only when status is unknown